agent/agent-bootstrap/src/main/java/io/opentelemetry/instrumentation/api/instrumenter/http/TemporaryMetricsView.java

```diff
@@ -0,0 +1,147 @@
+  private static final Set<AttributeKey> durationClientView = buildDurationClientView();
+  private static final Set<AttributeKey> durationServerView = buildDurationServerView();
+  private static final Set<AttributeKey> activeRequestsView = buildActiveRequestsView();
+
+  private static Set<AttributeKey> buildDurationAlwaysInclude() {
+    // the list of included metrics is from
+    // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attributes
+    Set<AttributeKey> view = new HashSet<>();
+    // START APPLICATION INSIGHTS MODIFICATIONS
+    // view.add(SemanticAttributes.HTTP_METHOD);
+    view.add(SemanticAttributes.HTTP_STATUS_CODE); // Optional
+    // view.add(SemanticAttributes.HTTP_FLAVOR); // Optional
+    // END APPLICATION INSIGHTS MODIFICATIONS
+    return view;
+  }
+
+  private static Set<AttributeKey> buildDurationClientView() {
+    // We pull identifying attributes according to:
+    // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attribute-alternatives
+    // We only pull net.peer.name and net.peer.port because http.url has too high cardinality
+    Set<AttributeKey> view = new HashSet<>(durationAlwaysInclude);
+    view.add(SemanticAttributes.NET_PEER_NAME);
+    view.add(SemanticAttributes.NET_PEER_PORT);
+    return view;
+  }
+
+  private static Set<AttributeKey> buildDurationServerView() {
+    // We pull identifying attributes according to:
+    // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attribute-alternatives
+    // With the following caveat:
+    // - we always rely on http.route + http.host in this repository.
+    // - we prefer http.route (which is scrubbed) over http.target (which is not scrubbed).
+    Set<AttributeKey> view = new HashSet<>(durationAlwaysInclude);
+    // START APPLICATION INSIGHTS MODIFICATIONS
+    // view.add(SemanticAttributes.HTTP_SCHEME);
+    // view.add(SemanticAttributes.HTTP_HOST);
+    // view.add(SemanticAttributes.HTTP_ROUTE);
+    // END APPLICATION INSIGHTS MODIFICATIONS
+    return view;
+  }
+
+  private static Set<AttributeKey> buildActiveRequestsView() {
+    // the list of included metrics is from
+    // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#attributes
+    Set<AttributeKey> view = new HashSet<>();
+    view.add(SemanticAttributes.HTTP_METHOD);
+    view.add(SemanticAttributes.HTTP_HOST);
+    view.add(SemanticAttributes.HTTP_SCHEME);
+    view.add(SemanticAttributes.HTTP_FLAVOR);
+    view.add(SemanticAttributes.HTTP_SERVER_NAME);
+    return view;
+  }
+
+  // START APPLICATION INSIGHTS MODIFICATIONS
+
+  static Attributes applyClientDurationAndSizeView(
```

Comment:
i don't get it why these two static methods have to be inside TemporaryMetricsView? it can be elsewhere right? 
Thought we agreed not to include this class.